cognite/neat/_rules/importers/_rdf/_inference2rules.py

```diff
@@ -455,37 +457,37 @@ def _create_classes_properties(
                         continue
                     property_id = remove_namespace_from_uri(property_uri)
                     self._add_uri_namespace_to_prefixes(property_uri, prefixes)
-                    if existing_prop := properties_by_id.get(property_id.casefold()):
+                    property_id_standardized = NamingStandardization.standardize_property_str(property_uri)
+                    if existing_prop := properties_by_id.get(property_id_standardized):
                         if not isinstance(existing_prop.instance_source, list):
                             existing_prop.instance_source = (
                                 [existing_prop.instance_source] if existing_prop.instance_source else []
                             )
                         existing_prop.instance_source.append(property_uri)
                         continue
                     else:
-                        properties_by_id[property_id.casefold()] = self._create_property(
+                        properties_by_id[property_id_standardized] = self._create_property(
                             read_properties, class_suffix, property_uri, property_id, prefixes
                         )
-                properties_by_class_suffix_by_property_id_lowered[class_suffix] = properties_by_id
+                properties_by_class_suffix_by_property_id[class_suffix] = properties_by_id
             if parent_suffix:
                 properties_by_id = {}
                 for property_uri, read_properties in shared_properties.items():
                     property_id = remove_namespace_from_uri(property_uri)
                     self._add_uri_namespace_to_prefixes(property_uri, prefixes)
-                    if existing_prop := properties_by_id.get(property_id.casefold()):
+                    property_id_standardized = NamingStandardization.standardize_property_str(property_uri)
+                    if existing_prop := properties_by_id.get(property_id_standardized):
                         if not isinstance(existing_prop.instance_source, list):
                             existing_prop.instance_source = (
                                 [existing_prop.instance_source] if existing_prop.instance_source else []
                             )
                         existing_prop.instance_source.append(property_uri)
                     else:
-                        properties_by_id[property_uri.casefold()] = self._create_property(
+                        properties_by_id[property_id_standardized] = self._create_property(
```

Comment:
Maybe nitpick: is it guarantee this key is valid? Maybe change this to dict.get as well

Reply:
Not sure I understand this one? This sets the value, so I don't understand why I need to use `.get`.

Reply:
nvm, forget about it. I was doing two things at the same time.